guides/Cloud_development/GCP.md

```diff
@@ -0,0 +1,33 @@
+## Google Cloud Platform
+
+**Google Cloud Platform** popularly referred to as GCP is a platform that offeres cloud computing services. It has been existing since 2008 till today.
+
+Some of its services includes; data storage, computing, google serch, Youtube, machine learninga and a few others. 
+
+GCP offers different other major services infrastructure; infrastructure, platform and serverless enviroments.
+
+**There are products offered by Google from this cloud platform and they are:**
```

Comment:
Why should a developer use GCP rather than any other platform? 😄

Reply:
😄Well basically GCP tools are far easier to use and quite cheap in most services.